circuits/circom/core/qv/processMessages.circom

```diff
@@ -164,50 +156,23 @@ template ProcessMessages(
 
     // Hash each Message to check their existence in the Message tree.
     var computedMessageHashers[batchSize];
+    var computedHashes[batchSize];
+    var chainHash[batchSize + 1];
+    chainHash[0] = inputBatchHash;
     for (var i = 0; i < batchSize; i++) {
```

Comment:
let's add comments here to explain what is happening inside this loop

Reply:
resolved